server/handler.go

```diff
@@ -9,16 +9,19 @@ import (
 	"github.com/labstack/echo"
 	"github.com/line/line-bot-sdk-go/linebot"
 	"github.com/pkg/errors"
-	"github.com/rs/zerolog/log"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	defaultCookieExpires = 60
+	uid                  = "123456789"
```

Comment:
should add `TODO: ...` comment that `uid` will be retrieved later once LIFF is implemented